src/backend/distributed/sql/downgrades/citus--10.0-1--9.5-1.sql

```diff
@@ -43,6 +43,14 @@ RENAME TO master_dist_shard_cache_invalidate;
 ALTER FUNCTION citus_drop_all_shards(regclass, text, text)
 RENAME TO master_drop_all_shards;
 
+#include "../udfs/citus_conninfo_cache_invalidate/9.5-1.sql"
+#include "../udfs/citus_dist_local_group_cache_invalidate/9.5-1.sql"
+#include "../udfs/citus_dist_node_cache_invalidate/9.5-1.sql"
+#include "../udfs/citus_dist_object_cache_invalidate/9.5-1.sql"
+#include "../udfs/citus_dist_partition_cache_invalidate/9.5-1.sql"
+#include "../udfs/citus_dist_placement_cache_invalidate/9.5-1.sql"
+#include "../udfs/citus_dist_shard_cache_invalidate/9.5-1.sql"
+
```

Comment:
I think we're leaving the citus_dist_shard_cache_invalidate functions in place here, which would break re-upgrade after downgrade. Would need some drop functions.

Reply:
actually, I'm wrong. We rename and then replace, so there are no old functions.